### DISCUSSION TITLE
Kernel upgrade from 5.4.xx to 5.10.xx

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#663] [General]
The latest kernel upgrade with `pacman -Syu` has renamed:
- `/boot/cmdline.txt` > `/boot/cmdline.txt.pacsave`
- `/boot/config.txt` > `/boot/config.txt.pacsave`

Get them back:
```
mv -f /boot/cmdline.txt{.pacsave,}
mv -f /boot/config.txt{.pacsave,}
```

**Comment by klassikmann** [comment]
Thank you.

**Comment by phreatic62** [comment]
I'm not sure I'm doing the update correctly, when I accept the default answers, nothing seems to get updated.


rAudio-Pi3:/boot  #  pacman -Syu
:: Synchronizing package databases...
 +R is up to date
 core is up to date
 extra is up to date
 community is up to date
 alarm is up to date
 aur is up to date
:: Starting full system upgrade...
warning: chromium: ignoring package upgrade (90.0.4430.212-1 => 96.0.4664.45-2)
:: Replace linux-raspberrypi with core/linux-rpi-legacy? [Y/n]
resolving dependencies...
looking for conflicting packages...
:: xorg-server and xf86-video-fbturbo-git are in conflict (X-ABI-VIDEODRV_VERSION). Remove xf86-video-fbturbo-git? [y/N]
error: unresolvable package conflicts detected
error: failed to prepare transaction (conflicting dependencies)
:: xorg-server and xf86-video-fbturbo-git are in conflict (X-ABI-VIDEODRV_VERSION>=25)
